src/components/Application.tsx

```diff
@@ -576,35 +583,40 @@ export class Application extends React.Component<Props, State> {
     })
   }
 
-  private deselectSelectedFeature(args: { ignoreTypes?: string[] } = {}) {
+  // Determine if the selected feature is an ignorable type
+  private isSelectedFeatureIgnorable(args: { ignoreTypes?: string[] } = {}) {
     args.ignoreTypes = args.ignoreTypes || []
 
     if (this.state.selectedFeature) {
       for (const type of args.ignoreTypes) {
         if (this.state.selectedFeature.properties.type === type) {
-          return // Don't deselect ignored types.
+          return true
         }
       }
-    }
+    };
```

Comment:
I added this semicolon because my Sublime JavaScript (Babel) plugin has some issue where it can't properly color the syntax on any code after this point. It's a strange issue, hopefully this semicolon doesn't cause issues?

Reply:
That's odd... I'm not having any issue without the semicolon in WebStorm. I left a comment about that line in the other PR, but I guess you can just ignore that if you can't find any way around it. It shouldn't cause any functional issues. It just seems like some kind of red flag if your syntax coloring breaks without it. Did you try closing all of the files and restarting your editor without that semicolon there? I've had times where my IDE will freak out for some reason until I restart it.